services/cantemo/types.go

```diff
@@ -13,9 +13,9 @@ type Video struct {
 	VideoCodec      string `json:"videoCodec"`
 	VideoFPS        string `json:"videoFPS"`
 	VideoResolution string `json:"videoResolution"`
-	VideoBitRate    int    `json:"videoBitRate"`
-	FieldOrder      string `json:"fieldOrder"`
-	PixelFormat     string `json:"pixelFormat"`
+	//VideoBitRate    int    `json:"videoBitRate"`
```

Comment:
```suggestion
```

Reply:
This one is commented out because it fails to parse the json in cases where `VideoBitRate` is `""` (aka no video) and it's not used anywhere.
I didn't want to remove it completely as we may want it back at some point, and it saves the guesswork "is it there or not"